packages/core/src/account/__tests__/simple.test.ts

```diff
@@ -10,23 +11,59 @@ import { getDefaultSimpleAccountFactoryAddress } from "../../utils/index.js";
 import { createSimpleSmartAccount } from "../simple.js";
 
 describe("Account Simple Tests", async () => {
+  const chain = sepolia;
   const dummyMnemonic =
     "test test test test test test test test test test test test";
   const owner: SmartAccountSigner =
     LocalAccountSigner.mnemonicToAccountSigner(dummyMnemonic);
+  const rundler = createRundler({
```

Comment:
in practice this is probably not what we want to use. we will want to use a pool like in https://github.com/wevm/anvil.js/blob/b09bd0ec8013a674dcc5dbe6885e12425bd1f823/examples/example-vitest/tests/utils.ts